react/components/admin/pages/Form/utils.ts

```diff
@@ -149,6 +149,7 @@ export const formatToFormData = (route: Route): RouteFormData => {
       ['metaTags', 'keywords'],
       route
     ).map(keyword => ({ label: keyword, value: keyword })),
+    metaTagRobots: pathOr('', ['metaTags', 'robots'], route),
```

Comment:
I didn't know about this `pathOr` from `Ramda` so I had to look at the [docs](https://devdocs.io/ramda/index#pathOr). 🆒 

The helper text for the Robots input mentions the default value is `index, follow`. I saw that for `pathOr`, the first argument is the default value, but here we're using an empty string `''`. Shouldn't we fallback to `'index, follow'`?

Reply:
Ah, just started reviewing https://github.com/vtex-apps/store/pull/569/files#diff-193c56439a9682751b1ab9811842225448c69a529534bd2c6ff40b029c89595a, and I think the default value is applied there!

Reply:
We can put the default value there also, but, if it comes as empty we will use in front the one from the store wrapper, I'm doing that to not force the user to store any unnecessary information